cmfive.php

```diff
@@ -182,7 +278,11 @@ function installCoreLibraries()
     // dist     : []
     // names    : 2pisoftware/cmfive-core
 
-    $composer_json = sketchComposerForCore();
+    if (empty($branch)) {
+        echo ("\nYou need to specify the target branch from core repository.\n");
```

Comment:
Shouldn't this function exit if a brach isn't specified?

Reply:
Dropped expectation of branch being filled, made this simply an advisory message.

Improved deliberation of "sketchComposerForCore()"